README.md

```diff
@@ -982,14 +983,18 @@ Please refer to the linked specific documents when looking for information regar
 
 [Top](#top)
 
-##<a name="section9"></a>Contact
+##<a name="section9"></a>Reporting issues and contact information
+There are several channels suited for reporting issues and asking for doubts in general. Each one depends on the nature of the question:
 
-Francisco Romero Bueno (francisco.romerobueno@telefonica.com) **[Main contributor]**
-<br>
-Fermín Galán Márquez (fermin.galanmarquez@telefonica.com) **[Contributor and Orion Context Broker owner]**
-<br>
-Germán Toro del Valle (german.torodelvalle@telefonica.com) **[Contributor]**
-<br>
-Iván Arias León (ivan.ariasleon@telefonica.com) **[Quality Assurance]**
+* Use [stackoverflow.com](http://stackoverflow.com) for specific questions about the software. Typically, these will be related to installation problems, errors and bugs. Development questions when forking the code are welcome as well. Use the `fiware-cosmos` tag.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/b152a9ce27595538f48d863b5aeaae27b005f2f7